.gitignore

```diff
@@ -161,3 +160,9 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 .idea/
+
+# VSCode settings (e.g. .vscode/settings.json containing personal preferred path to venv)
+.vscode/
+
+# macOS auto-generated file
+.DS_Store
```

Comment:
I approve this! It may be worthwhile to put these at the top of the file so that they're more prominent.